.pytool/Plugin/LineEndingCheck/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Line Ending Check Plugin
+
+This CiBuildPlugin scans all the files in a package to verify that the line endings are CRLF.
+
+## Configuration
+
+The plugin can be configured to ignore certain files.
+
+``` yaml
+"LineEndingCheck": {
+    "IgnoreFiles": []
+}
+```
+
+### IgnoreFiles
+
+An **optional** list of Unix style filename patterns used to exclude files from being checked.
+
+See [fnmatch](https://docs.python.org/3/library/fnmatch.html#module-fnmatch) for more details.

```

Comment:
is fnmatch still relevant?

Reply:
No. I removed this section.

Reply:
https://github.com/microsoft/mu_basecore/pull/123/commits/7211bef603a41285dd60cb6de852f7cfb1b04e8c